### PR TITLE
Added seat_apple_wallet_urls and seat_apple_wallet_urls to Seat

### PR DESCRIPTION
**Pull request by charklewis**

### CODE REVIEW

**Review by ASamarin04 — APPROVED** [review]
Looks good!

**Review by cakemanny — CHANGES_REQUESTED** [review]
Some code one has to write to use these would look like

```python
if seat.seat_apple_wallet_urls is not None:
    apple_wallet_url = seat.seat_apple_wallet_urls.get("apple_wallet_gen_url")
```
I think this is inconsistent with the design of `pyticketswitch`. I think it would be more consistent if `seat_apple_wallet_urls` was an object and the usage looked like :
```python
if seat.seat_apple_wallet_urls is not None:
    apple_wallet_url = seat.seat_apple_wallet_urls.apple_wallet_gen_url
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.8.8] - 2022-01-20
+### Added
+- Added seat_apple_wallet_urls and seat_apple_wallet_urls to Seat PR:(#105)
+
```
[the remark this answers is not in the document]
Comment (charklewis) [reply CHANGELOG.md:12]: ah yup, thanks dan! I have updated these https://github.com/ingresso-group/pyticketswitch/pull/105/commits/1626632c75125c1de552611bd56e1ee58c5e71bf

File: tests/test_seat.py [hunk 2]
```diff
@@ -33,6 +33,13 @@ def from_api_data(self):
             'seat_text_code': 'NEARTOILET',
             'separator': '-',
             'barcode': '0987654321',
+            'seat_apple_wallet_urls': {
+                'apple_wallet_gen_url': 'https://apple-wallet-api-dev.ticketswitch.io/generate_passes'
+            },
+            'seat_google_wallet_urls': {
+                'gpay_jwt_url': 'https://google-wallet-api-dev.ticketswitch.io/transactions/googlePayJWT',
+                'gpay_save_url': 'https://google-wallet-api-dev.ticketswitch.io/transactions/saveToGooglePay'
+            }
```
[the remark this answers is not in the document]
Comment (charklewis) [reply tests/test_seat.py:35]: Ah thank you again Dan! After some exploring I noticed that these tests were never actually getting run. I have fixed this now and also updated the value for the wallets.

**Comment by charklewis** [comment]
@ASamarin04 and @cakemanny I have updated the code based your feedback Dan. Let me know what you think. I wasn't able to test it against my local instance of my white label, so unsure 100% if it will work.

**Comment by charklewis** [comment]
I managed to test it locally by using `pip install -e /Users/charleslewis/Code/pyticketswitch`. Not sure if there is a better way but this seemed to work for me.

**Comment by charklewis** [comment]
This is the PR (https://github.com/ingresso-group/white-label/pull/3698) for the white label to consume this change. Note it is breaking as this version of pyticketswitch isn't deployed yet.

**Review by cakemanny — APPROVED** [review]
LGTM! :)

**Pull request merged**